src/v/kafka/server/handlers/produce.cc

```diff
@@ -40,12 +34,6 @@
 #include <boost/container_hash/extensions.hpp>
 #include <fmt/ostream.h>
 
-#include <chrono>
-#include <cstdint>
-#include <memory>
```

Comment:
std::unique_ptr is used in the .cc file

Reply:
true, but it is already included, so `<memory>` is not strictly required here

Reply:
but it's not included in produce.h